scripts/generate-sitemap.js

```diff
@@ -1,7 +1,10 @@
 const fs = require('fs')
 const globby = require('globby')
+const path = require('path')
 const prettier = require('prettier')
+const { cpuUsage } = require('process')
```

Comment:
This is probably not needed :)

Reply:
Yup ! (That was totally VsCode fault 😶 )